README.md

```diff
@@ -48,11 +54,11 @@ Set to `true` to start in Insert mode when opening files.
 
 Set to `true` to will make line numbers relative when not in Insert mode.
 
-#### `amVim.mimicVimSearchBehavior`
+#### `amVim.useSystemClipboard`
 
-`Boolean`, Default: `true`
+`Boolean`, Default: `false`
 
-Set to `false` to keep VSCode's keybinding when searching.
+Set to `true` to copy to and paste from the operating system clipboard.
```

Comment:
Just a nitpick, but I'm not sure if "operating system clipboard" is always going to be correct. I'm thinking of cases like Linux where there isn't really an operating system clipboard. I think X11 still has a middle click clipboard and a Ctrl-C (system) clipboard. Using the term "system clipboard" would probably be more accurate here.

Reply:
Ok sounds good, I had it as "system clipboard" before, and changed it to "operating system", thinking it would help less experienced folks. But let's use the simple one for now.

Reply:
Yeah, i guess the question is who the reader is going to be. If they're installing a Vim extension, and not even the most popular one in the marketplace, they're likely to be a fair bit more techy than the average developer. I was pondering over in the other PR if perhaps we should call this "yank" for maximum vimishness. Not sure. Either way it's a nitpick, i guess the majority of readers will just be happy to see anything documented.